File: DivisibleBy3and5OrNot.cpp
#include<iostream>
using namespace std;

class DivisibleBy3and5OrNot
{
    public:
      int No;

      DivisibleBy3and5OrNot(int iValue)
      {
          No = iValue;
      }

      bool Dividation()
      {
          if((No % 3 == 0)&&(No % 5 == 0))
          {
              return true;
          }
          else
          {
              return false;
          }
      }

};

int main()
{
    int iNo = 0;
    bool bRet = false;

    cout<<"Enter the number:\n";
    cin>>iNo;

    DivisibleBy3and5OrNot obj(iNo);

    bRet = obj.Dividation();
    if(bRet == true)
    {
        cout<<iNo<<" number is divisible by 3 and 5.\n";
    }
    else
    {
        cout<<iNo<<" number is not divisible by 3 and 5.\n";
    }

    return 0;
}

File: DivisibleBy3and5OrNot.java
import java.util.*;

class Mathematic
{
    public boolean Dividation(int No)
    {
        if((No % 3 == 0) && (No % 5 == 0))
        {
            return true;
        }
        else
        {
            return false;
        }
    }

}

class DivisibleBy3and5OrNot
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        int iNo = 0;
        boolean bRet = false;

        System.out.println("Enter the number.");
        iNo = sobj.nextInt();

        Mathematic mobj = new Mathematic();
        bRet = mobj.Dividation(iNo);
        if(bRet == true)
        {
            System.out.println(iNo + " number is divisible by 3 and 5.");
        }
        else
        {
            System.out.println(iNo + " number is not divisible by 3 and 5.");
        }


    }
}